content/manuals/build/building/cdi.md

```diff
@@ -0,0 +1,531 @@
+ Name:                  vendor1.com/device=foo
+ Automatically allowed: false
+GC Policy rule#0:
+ All:            false
+ Filters:        type==source.local,type==exec.cachemount,type==source.git.checkout
+ Keep Duration:  48h0m0s
+ Max Used Space: 658.9MiB
+GC Policy rule#1:
+ All:            false
+ Keep Duration:  1440h0m0s
+ Reserved Space: 4.657GiB
+ Max Used Space: 953.7MiB
+ Min Free Space: 2.794GiB
+GC Policy rule#2:
+ All:            false
+ Reserved Space: 4.657GiB
+ Max Used Space: 953.7MiB
+ Min Free Space: 2.794GiB
+GC Policy rule#3:
+ All:            true
+ Reserved Space: 4.657GiB
+ Max Used Space: 953.7MiB
+ Min Free Space: 2.794GiB
+```
+
+Now let's create a Dockerfile to use this device:
+
+```dockerfile
+# syntax=docker.io/docker/dockerfile-upstream:1.17-labs
+FROM busybox
+RUN --device=vendor1.com/device \
+  env | grep ^FOO=
+```
+
+Since [Dockerfile frontend v1.14.0-labs](../buildkit/dockerfile-release-notes.md#1140-labs),
+we can use the [`RUN --device` command](/reference/dockerfile.md#run---device)
+to request a device during a build. In this case we set `vendor1.com/device`
+device name which will request the first device available in the specification.
+In this case it will be `foo`.
+
+Now let's build this Dockerfile:
+
+```console
+$ docker buildx build .
+docker buildx build .
+[+] Building 0.4s (5/5) FINISHED                                                                                                                                                                docker:default
+ => [internal] load build definition from Dockerfile                                                                                                                                                      0.0s 
+ => => transferring dockerfile: 155B                                                                                                                                                                      0.0s
+ => resolve image config for docker-image://docker.io/docker/dockerfile-upstream:1.17-labs                                                                                                                0.1s 
+ => CACHED docker-image://docker.io/docker/dockerfile-upstream:1.17-labs@sha256:9187104f31e3a002a8a6a3209ea1f937fb7486c093cbbde1e14b0fa0d7e4f1b5                                                          0.0s
+ => [internal] load metadata for docker.io/library/busybox:latest                                                                                                                                         0.1s 
+ => [internal] load .dockerignore                                                                                                                                                                         0.0s
+ => => transferring context: 2B                                                                                                                                                                           0.0s 
```

Comment:
For some of these, it may make sense to also omit them, so that it's more clear what the relevant parts of the output are;


```suggestion
 => => transferring dockerfile: 155B                                                                                                                                                                      0.0s
 # ...
```